### DISCUSSION TITLE
[Release Vote] Vote for Release v0.6.0-rc1

### DISCUSSION THREAD

**Discussion by shidayang** [apache/amoro#2184] [Polls]
Hi Everyone,

I propose that we release the following RC as the official Amoro 0.5.1 release.

The commit ID is https://github.com/NetEase/amoro/commit/f1b911b65dea01f2881a646144c9ab9354a2126a

This corresponds to the tag: v0.6.0-rc1
https://github.com/NetEase/amoro/commits/v0.6.0-rc1
https://github.com/NetEase/amoro/tree/v0.6.0-rc1
Please check out, build, and test.

Please vote in the next 72 hours and add your test steps when voting.

**Comment by HuangFru** [comment]
-1, I found a critical bug : https://github.com/NetEase/amoro/pull/2186

**Comment by shidayang** [comment]
-1, Found a bug: #2187 #2195

**Comment by wangtaohz** [comment]
-1, found a bug working with S3, https://github.com/NetEase/amoro/issues/2192

**Comment by baiyangtx** [comment]
-1, for bug https://github.com/NetEase/amoro/issues/2190
